data/commodities.txt

```diff
@@ -867,19 +1007,32 @@ trade
 		"weaponized chickens"
 		"weapons-grade uranium"
 	commodity "void sprite parts"
+		"dismembered void sprites"
 		"void sprite parts"
 		"void sprite limbs"
-		"dismembered void sprites"
 	commodity "Military"
 		"ammunition"
 		"armor plating"
+		"armored cars"
+		"artillery"
 		"atmospheric craft"
 		"battlefield drones"
+		"bombers"
+		"bombs"
+		"bunker parts"
 		"camouflage fatigues"
 		"combat vehicles"
+		"explosives"
 		"field rations"
+		"fighter jets"
 		"firearms"
+		"flamethrowers"
 		"guided bombs"
+		"guided missiles"
+		"guns"
+		"heat detectors"
```

Comment:
```suggestion
		"heat detectors"
		"howitzers"
```
Since we have mortars

Reply:
I’ve got that, but I’m not sure how to make it co-authored by you.